cypress/support/pages/cart.pageObject.js

```diff
@@ -0,0 +1,54 @@
+import PageObject from '../PageObject';
+
+class CartPageObject extends PageObject {
+  assertProduct(productName) {
+    cy.get('.table')
+      .should('contain', productName);
+  }
+
+  ClickPlaceOrder() {
```

Comment:
I would like you to try an experiment here with arguments, then it will give you a lot more independence from your own methods.